images/linux/scripts/installers/oras-cli.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+################################################################################
+##  File:  oras-cli.sh
+##  Desc:  Installs ORAS CLI
+################################################################################
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/document.sh
+source $HELPER_SCRIPTS/install.sh
+
+# Determine latest ORAS CLI version
+ORAS_CLI_LATEST_VERSION_URL=https://api.github.com/repos/deislabs/oras/releases/latest
+ORAS_CLI_DOWNLOAD_URL=$(curl -s $ORAS_CLI_LATEST_VERSION_URL | jq -r '.assets[].browser_download_url | select(contains("linux_amd64.tar.gz"))')
+ORAS_CLI_ARCHIVE=$(basename $ORAS_CLI_DOWNLOAD_URL)
+
+# Install ORAS CLI
+cd /tmp
+download_with_retries $ORAS_CLI_DOWNLOAD_URL
+mkdir -p oras-install
+tar -xzvf $ORAS_CLI_ARCHIVE -C oras-install/
+mv oras-install/oras /usr/local/bin/
```

Comment:
Why do we need a temporary path?
`tar -zxvf $ORAS_CLI_ARCHIVE -C /usr/local/bin oras`

Reply:
Thank you, good point. Removed temporary path.